.github/workflows/release.yml

```diff
@@ -26,81 +26,81 @@ jobs:
-  #     contents: write
-  #   environment: release
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@v4
+  ssdlc:
+    needs: [release_please, build]
+    permissions:
+      # required for all workflows
+      security-events: write
+      id-token: write
+      contents: write
+    environment: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
 
-  #     - name: Install Node and dependencies
-  #       uses: mongodb-labs/drivers-github-tools/node/setup@v2
-  #       with:
-  #         ignore_install_scripts: true
+      - name: Install Node and dependencies
+        uses: mongodb-labs/drivers-github-tools/node/setup@v2
+        with:
+          ignore_install_scripts: true
 
-  #     - name: Load version and package info
-  #       uses: mongodb-labs/drivers-github-tools/node/get_version_info@v2
-  #       with:
-  #         npm_package_name: mongodb-client-encryption
+      - name: Load version and package info
+        uses: mongodb-labs/drivers-github-tools/node/get_version_info@v2
+        with:
+          npm_package_name: "@mongodb-js/zstd"
 
-  #     - name: actions/compress_sign_and_upload
-  #       uses: mongodb-labs/drivers-github-tools/node/sign_node_package@v2
-  #       with:
-  #         aws_role_arn: ${{ secrets.AWS_ROLE_ARN }}
-  #         aws_region_name: us-east-1
-  #         aws_secret_id: ${{ secrets.AWS_SECRET_ID }}
-  #         npm_package_name: mongodb-client-encryption
-  #         dry_run: ${{ needs.release_please.outputs.release_created == '' }}
-  #         sign_native: true
+      - name: actions/compress_sign_and_upload
+        uses: mongodb-labs/drivers-github-tools/node/sign_node_package@v2
+        with:
+          aws_role_arn: ${{ secrets.AWS_ROLE_ARN }}
+          aws_region_name: us-east-1
+          aws_secret_id: ${{ secrets.AWS_SECRET_ID }}
+          npm_package_name: "@mongodb-js/zstd"
+          dry_run: ${{ needs.release_please.outputs.release_created == '' }}
+          sign_native: true
 
-  #     - name: Copy sbom file to release assets
-  #       shell: bash
-  #       if: ${{ 'mongodb-client-encryption-6.1' == '' }}
-  #       run: cp sbom.json ${{ env.S3_ASSETS }}/sbom.json
+      - name: Copy sbom file to release assets
+        shell: bash
+        if: ${{ 'node-zstd' == '' }}
+        run: cp sbom.json ${{ env.S3_ASSETS }}/sbom.json
 
-  #     # only used for mongodb-client-encryption
-  #     - name: Augment SBOM and copy to release assets
-  #       if: ${{ 'mongodb-client-encryption-6.1' != '' }}
-  #       uses: mongodb-labs/drivers-github-tools/sbom@v2
-  #       with:
-  #         silk_asset_group: 'mongodb-client-encryption-6.1'
-  #         sbom_file_name: sbom.json
+      # only used for mongodb-client-encryption
+      - name: Augment SBOM and copy to release assets
+        if: ${{ 'node-zstd' != '' }}
+        uses: mongodb-labs/drivers-github-tools/sbom@v2
+        with:
+          silk_asset_group: "node-zstd"
```

Comment:
Where does this name come from, I just notice we have a "node-" prefix here but not the same on "mongodb-client-encryption", is it possible for us to use the npm package name?

Reply:
This was created for us a long time ago.  We could change it, but it would add a day to our testing process (this is the silk asset group, which has the overnight delay for silk to scan our repos).